### ISSUE TITLE
Add parameters : for exemple to disable forbidden 4XX

### ISSUE THREAD

**Issue by giraudvalentin** [ShogunPanda/fastify-http-errors-enhanced#3]
Hi

I was using your lib for two things : 
- Clarify errors in validation as well for client or server side.
- Make additionalProperties: false working by send an error to server side when result contain not declared fields in schemas

So, it work absolutely, BUT this lib do other things that i dont want.
- Add a stacktrace to validation error (not wanted)
- Refuse to send HTTP 4XX (only 500), Return 500 instead of a 422 due to : 
```
    // No validator, it means the HTTP status is not allowed
    const validator = validators[statusCode]

    if (!validator) {
      throw createError(StatusCodes.INTERNAL_SERVER_ERROR, validationMessagesFormatters.invalidResponseCode(statusCode))
    }
```
(this is due to the fact that i only declare 200 in schema, but no other http code, so may be the option is to only validate 200 for response ?)

I have tried to reimplement theses point by using hook but it add a lot of code.

I am thinking of forking this library to delete all theses features i didn't want, but think it may be cool if you add configurations parameters to the lib for this.

To sum up, i only want clear error object and respect of additionalProperties (or error), for the rest of features i didn't need, i just want an option for disable it.

Thank you

**Comment by ShogunPanda** [comment]
Hi!

Thanks for the very nice feedback.

About the second issue, which is allowing responses not defined in the schema, I'm adding a new option, called `allowUndeclaredResponses` that enables this behavior. Note that if a response is defined it will still be validated.
You can see and test the behavior in #4.

About the first issue, instead, I don't see the stack in my tests. Which kind of validation error are you throwing?
Can you please post a little repo example so I can fix it?

**Comment by ShogunPanda** [comment]
@giraudvalentin Any update on this?

**Comment by ShogunPanda** [comment]
@giraudvalentin I've released v2.2.0 which adds `allowUndeclaredResponses`. If you need anything else, feel free to reopne the issue. For now I'm closing this.

PS: Note that the package has been renamed to `fastify-http-errors-enhanced`. The use is the same.
